### ISSUE TITLE
Configuring the TCP port used to communicate with a LSP server

### ISSUE THREAD

**Issue by robertoaloi** [gtache/intellij-lsp#114]
What port is used by the LSP client to communicate to the server in case of a TCP transport?
Is it possible to configure the port?

**Comment by gtache** [comment]
It's not possible at the moment to use TCP without extending the plugin (it only supports stdin/out). I'll look into implementing TCP support in the next release possibly, if the plugin is stable enough.

**Comment by robertoaloi** [comment]
Yes, we realized that. And that's why we added support for STDIO to the [Erlang Language Server](https://github.com/erlang-ls/erlang_ls/) now. Still, we didn't manage to get the Erlang LS to work with IntelliJ via this plugin. A `Stream Closed` exception is the only thing visible in the IntelliJ logs. Since other developers seem to have issues to have their servers communicate with IntelliJ via this plugin (see for example #12 and #20 ) I wonder if it would be possible to display (maybe via a debug option) requests/responses in the logs.
